doc/rst/app_316_mc.rst

```diff
@@ -80,55 +81,55 @@ The master clock source is controlled by a mux which, in turn, is controlled by
 .. list-table:: Master Clock Source Selection
    :header-rows: 1
    :widths: 20 80
-  
+
    * - Value
      - Source
-   * - 0 
+   * - 0
      - Master clock is sourced from PhaseLink PLL
-   * - 1     
+   * - 1
      - Master clock is source from Cirrus Clock Multiplier
 
 The clock-select from the phaselink part is controlled via bit 7 of `PORT 8C`:
 
 .. list-table:: Master Clock Frequency Select
    :header-rows: 1
    :widths: 20 80
-  
+
    * - Value
      - Frequency
-   * - 0 
+   * - 0
      - 24.576MHz
-   * - 1     
+   * - 1
      - 22.579MHz
 
 DAC and ADC Configuration
-+++++++++++++++++++++++++
+-------------------------
 
 The board is equipped with a single multi-channel audio DAC (Cirrus Logic CS4384) and a single
-multi-channel ADC (Cirrus Logic CS5368) giving 8 channels of analogue output and 8 channels of 
+multi-channel ADC (Cirrus Logic CS5368) giving 8 channels of analogue output and 8 channels of
 analogue input.
```

Comment:
These part numbers are not correct for the XK-AUDIO-316-MC board. There are four stereo DACs (TI PCM5122) and two 4-channel ADCs (TI PCM1865).

Reply:
Thanks! fixed it.